bundler/lib/bundler/cli/gem.rb

```diff
@@ -56,6 +56,16 @@ def run
         options[:github_username]
       end
 
+     if github_username.nil? || github_username.empty?
+         homepage_uri = "TODO: Put your gem's homepage URL here."
+         source_code_uri = "TODO: Put your gem's public repo URL here."
+         changelog_uri = "TODO: Put your gem's CHANGELOG.md URL here."
```

Comment:
minor indentation issues.
```suggestion
      if github_username.nil? || github_username.empty?
        homepage_uri = "TODO: Put your gem's homepage URL here."
        source_code_uri = "TODO: Put your gem's public repo URL here."
        changelog_uri = "TODO: Put your gem's CHANGELOG.md URL here."
```